From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Fri, 21 Jan 2022 21:22:20 +0000
Subject: desktop usbview-pkexec

The Exec= line in the desktop file needs to invoke a wrapper script
which in turn invokes "pkexec /usr/bin/usbview", in order to get
DISPLAY passed through.
---
 usbview.desktop | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/usbview.desktop b/usbview.desktop
index 69f2bb8..d787d7c 100644
--- a/usbview.desktop
+++ b/usbview.desktop
@@ -2,7 +2,7 @@
 Name=USBView
 GenericName=USB Device Viewer
 Comment=View USB devices attached to system
-Exec=pkexec /usr/bin/usbview
+Exec=usbview-pkexec
 Icon=usbview
 Terminal=false
 Type=Application
